index.html

```diff
@@ -6,7 +6,8 @@
     <title>Części Samochodowe - wyszukiwarka codeVIN</title>
     <link rel="stylesheet" type="text/css" href="reset.css">
     <link rel="stylesheet" type="text/css" href="style.css">
-    <link rel="icon" type="image/png" sizes="16x16" href="favicon.ico/favicon-16x16.png">
+    <link rel="shortcut icon" href="favicon.ico/logo_pic.ico" type="image/x-icon">
+    <!--<link rel="icon" type="image/png" sizes="32x32" href="favicon.ico/favicon-16x16.png">-->
```

Comment:
Czy ten komentarz jest na wszelki wypadek? Jaka jest jego rola? ;) Pozdrówka!

Reply:
sloppy me. gonna fix it asap

Reply:
ok Push it on the same branch - your pull request should be correct than and I can merge it to develop ;)